include/PR/os_flash.h

```diff
@@ -4,6 +4,15 @@
 #include "ultratypes.h"
 #include "os_pi.h"
 
+#define FRAM_BASE_ADDRESS 0x08000000           // FRAM Base Address in Cart Memory
+#define FRAM_STATUS_REGISTER FRAM_BASE_ADDRESS // FRAM Base Address in Cart Memory
```

Comment:
ultralib doesn't have neither of those two defines, but it has `FLASH_START_ADDR`. It could be better to use that one instead. What do you think?